tools/mason/MasonPublish.chpl

```diff
@@ -267,6 +267,15 @@ proc dryRun(username: string, registryPath : string, isLocal : bool) throws {
     }
   }
   else {
+    const spacer = '------------------------------------------------------';
+    writeln('Checking Registry with ' + registryPath + ' path.');
+    var registryTest = registryPathCheck(registryPath, username, false);
+    writeln(spacer);
+    writeln('The current mason environment is:');
+    returnMasonEnv();
+    var reg = MASON_REGISTRY;
+    if reg.contains(('mason-registry', regUrl)) 
```

Comment:
I'm sorry, I lost sight of the bigger picture. We just want to verify that the MASON_REGISTRY isn't the default value, so we ought to check:

1. Does MASON_REGISTRY have 1 value?
2. If so, is that 1 value equal to the default registry?

so maybe it should be something like:

```chpl
var reg = MASON_REGISTRY
if reg.size == 1 {
  if reg[0] == ('mason-registry', regUrl)
    then writeln('   In order to use a local registry, ensure that MASON_REGISTRY points to the path.');
}
```